resources/views/vendor/log-viewer/dashboard.blade.php

```diff
@@ -13,7 +13,7 @@
       </ol>
       @stop
 @section('content')
```

Comment:
should not be part of the PR as does not contain any change except adding/removing whitespaces

Reply:
reverted